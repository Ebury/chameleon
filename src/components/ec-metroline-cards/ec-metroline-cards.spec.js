import { mount } from '@vue/test-utils';
import EcMetrolineCards from './ec-metroline-cards.vue';

describe('EcMetrolineCards', () => {
  function mountMetrolineCards(mountOpts) {
    return mount(EcMetrolineCards, { ...mountOpts });
  }

  it('should render as expected', async () => {
    const wrapper = mountMetrolineCards({
      slots: {
        default: '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>',
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
