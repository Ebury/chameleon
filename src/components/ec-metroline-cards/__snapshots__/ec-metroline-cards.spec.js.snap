// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EcMetrolineCards should render as expected 1`] = `
<div
  class="ec-metroline-cards"
  data-test="ec-metroline-cards"
>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  </p>
</div>
`;
